src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/DroolBase.java

```diff
@@ -76,6 +79,7 @@ public void setUp() {
         KieContainer kieContainer = ks.getKieClasspathContainer();
         this.kieSession = kieContainer.newStatelessKieSession("role-assignment-validation-session");
         this.kieSession.setGlobal("DATA_SERVICE", retrieveDataService);
+        this.kieSession.setGlobal("logger", RULE_LOGGER);
```

Comment:
Please remove it if not using in any drool file.

Reply:
It has been already removed.